canonical/snapcraft.io
# Remove list of snaps a user is a collaborator on

# To be rebased after https://github.com/canonical/snapcraft.io/pull/4286 is merged

## Done
- Remove fetching and storing *all* brand stores in the session

## How to QA
- Login to https://snapcraft-io-4287.demos.haus/
- Ensure brand store admin still works as before
- Go to your list of snaps
- Click "Register a snap name"
- You should see a dropdown with a list of available stores
- Attempt to register a name in one of the stores
- All screens you see in this flow should contain the dropdown with the list of stores you have publisher rights to.

## Issue / Card
Fixes https://github.com/canonical/snapcraft.io/issues/3838
Fixes https://github.com/canonical/snapcraft.io/issues/4277
